owners/src/ownership/owners_check.js

```diff
@@ -72,6 +80,7 @@ class CheckRun {
     return checkRun;
   }
 }
+CheckRun.prototype.textLimit = 65535;
```

Comment:
I suggest hoisting `65535` to a const at the top of the file and adding a comment that links to the Github API docs (even though they don't document the upper limit).

Reply:
It's done this way so it can be readily stubbed out in testing code. I'm not aware of a clean way to do so in the case of a module constant used within the module. I'm going add the comment and submit for now, but if you have a different preference here later I will address in a follow-up.

Reply:
I've established a module constant, and assign it to the prototype field, so that it can still be stubbed. And I've added a comment. Thanks! :+1:

Reply:
> I've established a module constant, and assign it to the prototype field, so that it can still be stubbed.

That's exactly what I meant 👍